packages/core/src/shared/telemetry/vscodeTelemetry.json

```diff
@@ -240,6 +240,11 @@
             "type": "string",
             "description": "Type of chat command name executed"
         },
+        {
+            "name": "codewhispererSuggestedTokens",
```

Comment:
when are these telemetry definitions going to be upstreamed to https://github.com/aws/aws-toolkit-common ? many of these have been here for 6+ months.

Reply:
this particular event is here because we were collecting `codewhispererUserInputDetails` for VS Code debugging, agreed we should remove it when it's no longer needed.

Reply:
@leigaol do you know if we still need this data?

Reply:
You can remove it now.